Added new question for HR

Kathuria opened the pull request.

Kathuria committed:
Added new query for HR

alexakasanjeev commented:
hi @Kathuria  Thank you so much for the your contribution 🎉

sanjeevyadavIT commented on README.md:
```diff
@@ -19,6 +19,7 @@ At the time of joining, the HR potrays the company like it is match made in Heav
 | 10 | Usually help half or whole financially for IT events and certifications?|
 | 11 | Can you describe the culture of the company?|
 | 12 | How easily the upper management is approachable for suggestion or queries?|
+| xx | Does your organisation is open for on-site opputunities and Can I get this oppurtunity on starting of day of an organisation? |
```
- In place of xx, mention the question number in chronological order
- their is a typo in spelling "opportunity", please fix that
- Second part of the question could be modified to `if yes, do you give these opportunity to new comers`?

Kathuria replied:
Thanks for the update. Changes are Done, Please Review it again

sanjeevyadavIT reviewed (changes requested):
There are some changes requested, please look into it

and if you are having trouble, making new changes, let me know, would be happy to help :smile:

Kathuria committed:
Incorporating review comments

Incorporating Details:
a) Replaced xx with question number in chronological order
b) Typo in spelling "opportunity"
c) Updated question
